### ISSUE TITLE
machine should not go into wait-for-installation if sum tool cannot change boot order

### ISSUE THREAD

**Issue by mwennrich** [metal-stack/go-hal#3]
e.g. from machine 32c8ee00-aa08-11e9-8000-ac1f6b7b771e:

```
INFO[08-20|05:35:04] wipe                                     caller=wipe.go:29
INFO[08-20|05:35:05] wipe                                     disk=/dev/sda message="start fast deleting of existing data" 
(...)
Success formatting namespace:1
INFO[08-20|05:35:23] firmware                                 is=2 caller=sum.go:164
Supermicro Update Manager (for UEFI BIOS) 2.4.0 (2019/12/06) (x86_64)
Copyright(C) 2013-2019 Super Micro Computer, Inc. All rights reserved.
DBUG[08-20|05:35:23] lldp                                     detectedNeighbor="Name:stg-kkw701-r01leaf02 Desc:Cumulus Linux version 3.7.12 running on Accton AS7712-32X Chassis:Mac:80:a2:35:e1:4c:9c Port:Mac:80:a2:35:e1:4c:a3" caller=lldpclient.go:71

********************************<<<<<ERROR>>>>>*********************************

ExitCode                = 80
Description             = Node product key is not activated.
Program Error Code      = 260.14
Error message:
        One of the node product key (OOB, SUM or DCMS) should be activated to
    execute this task.
        DCMS node product key is not activated.
        SUM node product key is not activated.
        OOB node product key is not activated.

********************************************************************************
WARN[08-20|05:35:27] BIOS updates for this machine type are intentionally not supported, skipping ConfigureBIOS error="unable to get BIOS configuration via:sum -c GetCurrentBiosCfg --file biosCfg.xml: exit status 80" caller=bios.go:20

INFO[08-20|05:35:28] event                                    event=Waiting message="waiting for installation" caller=event.go:62

```

**Comment by kolsa** [comment]
Will be fixed by #19 and https://github.com/metal-stack/metal-hammer/pull/39

**Comment by majst01** [comment]
both prs are merged, closing
